<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>10</horstretch>
    <verstretch>10</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48,48,48);
</string>
  </property>
  <property name="iconSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="heat_btn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>210</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#heat_btn{
border-image: url(:/Pictures/Qt menus/fire_on.gif);
background-repeat:none;
}
#heat_btn:pressed{
border-image: url(:/Pictures/Qt menus/fire_off.gif);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>81</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#start_btn{
border-image: url(:/Pictures/Qt menus/btn_off.gif);
background-repeat:none;
}
#start_btn:pressed{
border-image: url(:/Pictures/Qt menus/btn_on.gif);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="chart_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Pictures/Qt menus/mechanism.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>-10</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="s.qrc">:/Pictures/logo_label.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="analogclockwindow" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Purisa&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menud">
    <property name="title">
     <string>d</string>
    </property>
   </widget>
   <addaction name="menud"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionss">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Pictures/ss.png</normaloff>:/Pictures/ss.png</iconset>
   </property>
   <property name="text">
    <string>ss</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Pictures/ss.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>analogclockwindow</class>
   <extends>QWidget</extends>
   <header>analogclockwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>heat_btn</tabstop>
  <tabstop>start_btn</tabstop>
 </tabstops>
 <resources>
  <include location="s.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chart_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_chart_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
